src/events.rs

```diff
@@ -312,3 +312,11 @@ pub enum VirtualKeyCode {
     WebStop,
     Yen,
 }
+
+#[derive(Default, Debug, Clone, Copy)]
+pub struct Mods {
+  pub shift: bool,
+  pub ctrl: bool,
+  pub alt: bool,
+  pub supr: bool
```

Comment:
Other possible name: internally, xkbcommon calls it the "logo key", which is actually a pretty accurate name